README.md

```diff
@@ -333,6 +333,9 @@ or install via [lowmad](https://github.com/bangerang/lowmad):
 $ lowmad install https://github.com/YusukeHosonuma/SwiftPrettyPrint.git
 ```
 
+**Note:**
+If you already install version 1.1.0 or older via [lowmad](https://github.com/bangerang/lowmad), please remove scripts manually before update. (e.g. `rm /usr/local/lib/lowmad/commands/YusukeHosonuma-SwiftPrettyPrint/swift_pretty_print.py`)
```

Comment:
It says `version 1.1.0`, but I think it is hard to understand what version it is referring to.

Reply:
Clarified.

```suggestion
If you already installed 1.1.0 or older version of SwiftPrettyPrint via [lowmad](https://github.com/bangerang/lowmad), please remove scripts manually before update. (e.g. `rm /usr/local/lib/lowmad/commands/YusukeHosonuma-SwiftPrettyPrint/swift_pretty_print.py`)
```